src/fiber/graph.rs

```diff
@@ -127,14 +128,11 @@ impl ChannelInfo {
         &self.udt_type_script
     }
 
-    // Whether this channel is explicitly disabled in either direction.
-    // TODO: we currently deem a channel as disabled if one direction is disabled.
-    // Is it possible that one direction is disabled while the other is not?
     pub fn is_explicitly_disabled(&self) -> bool {
         match (&self.update_of_node2, &self.update_of_node1) {
-            (Some(update1), _) if !update1.enabled => true,
-            (_, Some(update2)) if !update2.enabled => true,
-            _ => false,
+            (Some(update1), _) if update1.enabled => false,
+            (_, Some(update2)) if update2.enabled => false,
+            _ => true,
```

Comment:
this fn will return true when `update_of_node1` and `update_of_node2` are None, this doesn't seem to fit the intent of the fn name, could you check it again?